docs/pre.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
Looks like SDK uses this pre hook to copy over the `spec` directory from each module in `x/`. The issue is our layout is slightly different since `apps` and `light-clients` are nested a layer deeper. Only solo machine has a `spec` repo for the light clients

In general I think we could improve upon the usage of the `spec` repo #1. Might be a good time to discuss this

We should add a comment on what this pre script is doing

Reply:
yea, that makes sense, i just copied it over from the sdk repo for now bc the Makefile ran it. @crodriguezvega is taking over this PR, so it's actually still Work in Progress.